### DISCUSSION TITLE
Reversible ESC for plane configuration

### DISCUSSION THREAD

**Discussion by Samureimer** [iNavFlight/inav#7067] [Q&A] [answered]
Hi.
I have a plane build where I took the only ESC I had left.
It is a Rcharlance 80A branded ESC from Ebay.
![MX03927-02-3](https://user-images.githubusercontent.com/5878260/120602262-29ee3380-c44b-11eb-90c5-d3249c9948c4.jpg)
(http://image-us.bigbuy.win/upload/image/assistant_no_logo/MX03927-02/MX03927-02-3.jpg)

The ESC is configured as a reversible ESC meaning the neutral point seems to be at 1500.
I have tried configuring the ESC but nothing seems to work.

My question is:
If you have a non-configurable ESC configured for forward and reverse direction, could you configure INAV to use the ESC only in the forward direction?

Best regards, Mark.

**Comment by pawel-spychalski** [comment, marked as answer]
yes, in not-DSHOT protocol this is doable by adjusting min_command setting. But then all other throttle settings should also be adjusted. In general, I suggest getting a ESC designed for airplanes

**Reply by Samureimer** [reply]
Thank you for the answer! :)

Another ESC would definitely be the best solution.
Regardless, I'll try your suggestion later today.

**Reply by Samureimer** [reply]
Hi Pawel and whoever might be interested in the solution.
I set the _min_command_ parameter to 1500 and it just works!

Thanks again :)
